dashboard/app/models/census/state_cs_offering.rb

```diff
@@ -18,16 +18,19 @@ class Census::StateCsOffering < ApplicationRecord
   belongs_to :school, foreign_key: :state_school_id, primary_key: :state_school_id, required: true
 
   validates_presence_of :course
-  validates :school_year, presence: true, numericality: {greater_than_or_equal_to: 2016, less_than_or_equal_to: 2030}
+  validates :school_year, presence: true, numericality: {greater_than_or_equal_to: 2015, less_than_or_equal_to: 2030}
 
   SUPPORTED_STATES = %w(
     AR
     CA
+    CT
```

Comment:
Three states included deliberately?

Reply:
Yup - to avoid merge conflicts with the numerous case conditions, I've been creating each state branch from the previous state branch that I implemented.  I merge each feature branch into staging in the order that I created the feature branch.